src/Makefile.am

```diff
@@ -463,6 +463,7 @@ endif
 libbitcoin_wallet_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) $(BOOST_CPPFLAGS) $(BDB_CPPFLAGS) $(SQLITE_CFLAGS)
 libbitcoin_wallet_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 libbitcoin_wallet_a_SOURCES = \
+  blockfilter.cpp \
```

Comment:
In 050716c48d703424dc94fde9d7f987b55cbff6a9 "Use wallet filter during rescan if available."

I feel like there's a better way to do this, but I'm not familiar enough with build systems to suggest one. Perhaps @fanquake might have an idea?

Reply:
yes this felt kind of wrong but I didn't see another way to achieve the result. suggestions definitely wanted.